backend/src/ml_space_lambda/authorizer/lambda_function.py

```diff
@@ -166,7 +164,7 @@ def lambda_handler(event, context):
                     and "username" in path_params
                     and request_method in ["GET", "PUT", "DELETE"]
                 ):
-                    if Permission.ADMIN in user.permissions:
+                    if IS_ADMIN:
```

Comment:
Per our discussion in standup it sounds like we'll just move the IS_ADMIN check to the beginning of the authorizer

Reply:
> I think we should hold off on bigger authorizer changes of short circuiting if they’re an admin. Some places in the authorizer set a variable on response_context (primarily the project name) and short circuiting evaluation would prevent that. I don’t immediately have a great solution for that either…